### DISCUSSION TITLE
Welcome to Procyon Discussions!

### DISCUSSION THREAD

**Discussion by george-norton** [george-norton/procyon#1] [Show and tell]
## 👋 Welcome!
Feel free to share your build pictures or ask questions here.

**Comment by jwgpro** [comment]
Hey, just wanted to say that I really appreciate this project. It's a great liberation knowing that I'm not dependent on the supply or design of a proprietary implementation. I've ordered some 57x80 units from JLC, experimented with surfaces and got them working, and am in the process of building them into some keyboards for a project called Devotion laptop, which will also be free/libre hardware. I'll be sure to post pics when I'm done!

**Reply by george-norton** [reply]
Amazing! Looking forward to seeing it.

**Comment by chrisoyng** [comment]
Which settings and options to do on JLCPCB? What layers do I export for the gerbers?

**Reply by george-norton** [reply]
There are kicad plugins, such as Bounis JLC tools which take care of everything. You need to manually check the orientation of some parts (particulary the maxtouch IC). The render JLC shows of fhe MaxTouch IC is confusing, its best to spin it around in their viewer just to sanity check it.
